### ISSUE TITLE
Insufficient Contrast in project descriptions in the footer

### ISSUE THREAD

**Issue by brtrx** [openaustralia/publicwhip#1086]
Descriptions of other projects in the footer don't have enough contrast to be readable until moused over.

![image](https://cloud.githubusercontent.com/assets/8436356/13802529/29853908-eb3d-11e5-80eb-2c3d638ba769.png)

Using the test available here: http://webaim.org/resources/contrastchecker/ it's clear this colour choice isn't WCAG AA or AAA compliant.

![image](https://cloud.githubusercontent.com/assets/8436356/13802549/4a80417a-eb3d-11e5-914d-25b3bb919417.png)

## My Suggestion

Changing the font colour to #95989E would still keeps the intended "knocked-back” colour scheme of the description text, to avoid pulling focus, and is AA compliant.

![image](https://cloud.githubusercontent.com/assets/8436356/13802582/ae2e725a-eb3d-11e5-8980-a5dad6926b58.png)

A lighter colour is required to achieve AAA compliance. IMHO, #adb3bf looks ok.

![image](https://cloud.githubusercontent.com/assets/8436356/13802568/81126330-eb3d-11e5-9d91-ad523feac77e.png)

**Comment by brtrx** [comment]
Note: this is related to, but distinct from #877

**Comment by henare** [comment]
Thanks for logging this @brtrx, it's a great bug report :+1: Assigning to @equivalentideas to review.

I'll just point out that from what I can tell these links have resulted in a total of 1 click to our other sites in the last month. It was to Right To Know and I suspect it may have been when testing this issue :wink:

**Comment by brtrx** [comment]
Fair enough, similar to most footers, I'd imagine.
IMO they serve as much or more of an orientation function, as they do a connection function.
In other words, making them more legible is a good move in terms of helping people quickly get a sense of what sort of stuff you do, even if they never follow (or even roll over) those links.

**Comment by equivalentideas** [comment]
Thanks for this @brtrx 🌠  I've change it and also removed the link to Election Leaflets since we're not running it for the foreseeable future.

![screen shot 2016-08-12 at 3 33 26 pm](https://cloud.githubusercontent.com/assets/1239550/17613586/5731ec5e-60a2-11e6-9a96-1d5eec13f3e7.png)
